### DISCUSSION TITLE
Relative Phase between two gaussian pulses

### DISCUSSION THREAD

**Discussion by HollyHuddle** [BLAST-WarpX/warpx#4301] [Simulation Inputs] [answered]
Hi there, just a quick question in case I've missed it somewhere in the documentation. Is there a variable which allows you to set a relative phase difference between two gaussian pulses? I'm just defining it with the parse_field_function option for now.

**Comment by RemiLehe** [comment, marked as answer]
Hi,

If you use the `"Gaussian"` profile, than you can use the parameter `phi0` to tune the phase, as explained here:
https://warpx.readthedocs.io/en/latest/usage/parameters.html#laser-initialization

On the other hand, if you instead use the `"parse_field_function"` profile, then you will need to include the phase explicitly in your mathematical formula. (If this is not clear, feel free to post your input script, and we can tell you how to modify it.)
